src/layout/layout.rs

```diff
@@ -1153,29 +1156,29 @@ mod tests {
         let fixed_size_array = [Constraint::Min(0)];
         let layout = Layout::new(Direction::Horizontal, fixed_size_array);
         assert_eq!(layout.direction, Direction::Horizontal);
-        assert_eq!(layout.constraints, [Constraint::Min(0)]);
+        assert_eq!(&*layout.constraints, [Constraint::Min(0)]);
```

Comment:
This looks pretty ugly - perhaps rephrase as `smallvec![Constraint::Min(0)]`

Reply:
Not sure I agree, (to my eyes) this is pretty standard deref coercion but I'll switch if people really don't like it.

Reply:
Ugly mostly because test code should look simple. When you bring in deref coercion just to test equality, that's no longer simple.

Reply:
Ah, ok I agree with that.

Reply:
Actually SmallVec just needs to fix its partialeq impls: https://github.com/servo/rust-smallvec/pull/358